chia/types/mempool_submission_status.py

```diff
@@ -0,0 +1,28 @@
+from dataclasses import dataclass
+from typing import Dict, Optional, Union
+
+from chia.types.mempool_inclusion_status import MempoolInclusionStatus
+from chia.util.ints import uint8
+from chia.util.streamable import Streamable, streamable
+
+
+@streamable
+@dataclass(frozen=True)
+class MempoolSubmissionStatus(Streamable):
```

Comment:
Note that I'm not converting the `sent_to` members in `TradeRecord` and `TransactionRecord` to use this struct, because that would change the DB, the Trade protocol, and the blockchain protocol.